http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -29,7 +33,22 @@ services:
       PIP_TARGET: "${NOTEBOOKS_DIR}/packages"
       R_LIBS: "${NOTEBOOKS_DIR}/packages"
     env_file:
-      - /etc/google_application_credentials.env
-      - /etc/custom_env_vars.env
+      - /var/google_application_credentials.env
+      - /var/custom_env_vars.env
     # See https://docs.docker.com/engine/reference/run/#user-memory-constraints
     mem_limit: ${MEM_LIMIT}   # hard limit on memory consumption by the container
+  welder:
+    ports:
+      - "8080:8080"
+    networks:
+      - app_network
+  crypto-detector:
+    network_mode: "service:jupyter"
```

Comment:
TIL about `network_mode: service:xxx`. I guess this means it shares the network stack with another container?

Reply:
yea, this is for the cryto-detector container to share network with the app container